low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -84,55 +80,14 @@ def get_command(desc: str, argv: dict):
     @staticmethod
     def add_args(parser: str, cls: str, name: str):
         """Add Command args for parsing."""
-        parsers = parser.add_parser(cls.name, help='setup %s' % name)
+        parsers = parser.add_parser(cls.name, help='%s' % cls.__doc__)
         parsers.add_argument('args', nargs='*', default=[], help='args')
         parsers.set_defaults(command=cls)
 
-    @staticmethod
-    def _send_command(command: str, fail_on_error=True):
-        # Note: This function uses subprocess to execute commands, scripts which are not possible to execute
-        # through any python routines available. So its usage MUST be limited and used only when no other
-        # alternative found.
-        process = subprocess.Popen(
-            command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        response, error = process.communicate()
-        if error is not None and \
-        len(error) > 0:
-            print("command '%s' failed with error\n%s" % (command, error))
-            if fail_on_error:
-                sys.exit(1)
-            else:
-                return str(error)
-        return str(response)
-
-    @staticmethod
-    def _call_script(script_dir: str, args: list):
-        script_args_lst = [script_dir]+args
-        subprocess.call(script_args_lst, shell=False)
-
-
-class SetupCmd(Cmd):
```

Comment:
Why it is removed?

Reply:
Becoz no further python api to invoke for sspl_setup, which is a CLI itself. Looks fine